support/cas-server-support-couchdb-ticket-registry/src/main/java/org/apereo/cas/config/CouchDbTicketRegistryConfiguration.java

```diff
@@ -30,10 +31,13 @@
     @Autowired
     private CasConfigurationProperties casProperties;
 
+    @Autowired
+    private ObjectMapperFactory objectMapperFactory;
```

Comment:
Probably should use a qualifier here to make overrides easier.

Reply:
Done